spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 require 'capybara/rspec'
 require 'rails_helper'
 require 'factory_girl_rails'
+require 'support/factory_girl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.